pallets/runtime/src/utils.rs

```diff
@@ -15,263 +14,3 @@ pub trait Trait: frame_system::Trait + BalancesTrait + pallet_session::Trait {
         v: <Self as pallet_session::Trait>::ValidatorId,
     ) -> Self::AccountId;
 }
```

Comment:
I wonder whether `utils.rs` is a good place for this trait. Especially given that the only function in this module has just been removed. Options? Either rename the file to suit the purpose of the trait or move it outside the runtime pallet, perhaps to `common::traits`.

Reply:
Actually, we no longer use that function at all. This file can be completely deleted but I think it's best done in a separate PR since a lot of files have to be modified to remove traces of this file.

Reply:
ps. We might need this function again when we work on the module fees task (MESH-688). Let's decide what to do with this file then.